### DISCUSSION TITLE
Getting Alarm after detection

### DISCUSSION THREAD

**Discussion by hasnatahmed650** [ultralytics/yolov5#6861] [Q&A]
Hi, hope all are fine. My question is that what amendment should I do for ringing Alarm after successful detection. Here is my problem: I have detected successfully a pest and a bounding box is appear. Now I want to ring a alarm after it is detected. So what will be the solution? I am looking forward to your suggestions.

**Comment by adelchellabi** [comment]
Hello,do you find the solution?
Because i need it thank you

**Comment by hasnatahmed650** [comment]
Yes

**Comment by adelchellabi** [comment]
Hello
Please can you share with me the solution?
Thank you
Best regards

**Comment by hasnatahmed650** [comment]
Can you please let me know about your project?
Alternatively you can contact +923495769027

**Reply by 1Yleng** [reply]
Hello，now I need this solution, please share your idea with me.
Thanks.

**Comment by ExtReMLapin** [comment]
It's not especially related to yolov5, it's just about programming, look for the detected classes in python.

**Reply by 1Yleng** [reply]
I want to achieve the effect:
Voice alarm when the target in the video is detected.

**Comment by hasnatahmed650** [comment]
Hello! Yes I can help you out with this!

**Reply by 1Yleng** [reply]
How can I contact you?
Is email OK?

**Comment by hasnatahmed650** [comment]
Whatsapp?

**Reply by 1Yleng** [reply]
Twitter ok?
@firework_Yleng

**Reply by 1Yleng** [reply]
Hello？
I need your help, please.
